social_website/views.py

```diff
@@ -522,3 +525,15 @@ def signup_view(request, template_name='social_website/signup.html',
 
     return TemplateResponse(request, template_name, context,
                             current_app=current_app)
+
+def save_email_and_get_data(request):
+    email = request.GET.get("email", "")
+    filename = 'connector_emails.txt'
+    if os.path.exists(filename):
```

Comment:
You do need to check for file's existence, opening in append mode will create the file if it does not exist, more information: https://docs.python.org/2.7/library/functions.html#open

Reply:
Need manual effort to create the file every time when server starts. to avoid that i write the code in such a way that if file exists we will append the emailId at the end of the file, if not exists we will create the new file to write.
`    if os.path.exists(filename):
        append_write = 'a'  # append if already exists
    else:
        append_write = 'w'  # make a new file if not:
    file_object = open(filename, append_write)`